core/src/main/java/io/questdb/cairo/AbstractIntervalDataFrameCursor.java

```diff
@@ -63,6 +63,16 @@ protected static long search(ReadOnlyColumn column, long value, long low, long h
                 low = mid + 1;
             else if (midVal > value)
                 high = mid;
+            else if (midVal > 0) {
+                // In case of multiple equal values, find the first
+                int offset = 0;
+                long prevVal;
+                do {
+                    prevVal = column.getLong((mid + --offset) * 8);
```

Comment:
the getLong() arg can't be negative, there no check for it, which is what likely has caused CI to crash